packages/liveblocks-core/src/room.ts

```diff
@@ -1156,6 +1157,23 @@ export function createRoom<
     comments: makeEventSource<CommentsEventServerMsg>(),
   };
 
+  async function streamFetch(authTokenOrPublicApiKey: string, roomId: string) {
+    const baseUrl = new URL(config.liveblocksServer);
+    baseUrl.protocol = "https";
```

Comment:
Unrelated to this PR, but I'm doing similar hacking elsewhere! Should we redefine the "liveblocks server" config to be a value like `https://api.liveblocks.io` and not `wss://api.liveblocks.io/v7`?

Reply:
Answered on slack

Reply:
Created an issue for this to track: #1230